plugins/inputs/kube_inventory/kube_inventory.go

```diff
@@ -140,6 +145,57 @@ func (ki *KubernetesInventory) convertQuantity(s string, m float64) int64 {
 	}
 	return int64(f * m)
 }
+func (ki *KubernetesInventory) LoadJSON(url string, v interface{}) error {
+	var req, err = http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
```

Comment:
Please define variables implicitly if possible and annotate the errors for better debugging.
```suggestion
	req, err := http.NewRequest("GET", url, nil)
	if err != nil {
		return fmt.Errorf("creating request failed: %w", err)
	}
```